Java/src/com/company/Serialization.java

```diff
@@ -0,0 +1,53 @@
+package com.company;
+
+import java.io.*;
+
+/**
+ * Created by Vova Moskalenko on 04.06.2017.
+ */
+public class Serialization implements java.io.Serializable {
+
+    String name = "Marcus";
+    String city = "Roma";
+    int born = 121;
+
+    public static void main(String[] args) {
+
+        Serialization person = new Serialization();
+        //serialization
+        try {
+            FileOutputStream fileOut =

```

Comment:
File input and output are redundant here. You should display exactly serialization and deserialization in this example. For instance, you can use JSON serialization and deserialization as in JavaScript example.